src/hotspot/cpu/x86/x86.ad

```diff
@@ -1861,6 +1861,14 @@ const bool Matcher::match_rule_supported_vector(int opcode, int vlen, BasicType
         return false;
       }
       break;
+    case Op_SignumVD:
+    case Op_SignumVF:
+      if (UseAVX < 1 ||
+          (size_in_bits == 512 && !VM_Version::supports_avx512dq()) ||
+          (size_in_bits == 256 && !VM_Version::supports_avx2())) {
```

Comment:
May I ask why do we need avx2 for 256-bit, thanks.

Reply:
Current instruction uses VPOR operations which works over 256 bit integral vectors only for targets supporting AVX2. 
VPOR is needed to compute a combined mask for NaN , -0.0 or 0.0. In all these case original source lanes should be returned back.